Packs/ArcherRSA/ReleaseNotes/1_1_3.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### RSA Archer v2
+- Fixed an issue where using *sorce* field in *application id 433* in command **archer-create-record** to create a record.
```

Comment:
```suggestion
- The argument *fieldsToValues* is now accepting `Values List` fields without `[]` e.g:
`fieldsToValues={"Priority":"High"}` instead of `fieldsToValues={"Priority":"[High]"}`
```

Reply:
done